libminifi/CMakeLists.txt

```diff
@@ -125,6 +125,8 @@ endif()
 list(APPEND LIBMINIFI_LIBRARIES yaml-cpp ZLIB::ZLIB concurrentqueue RapidJSON spdlog cron Threads::Threads gsl-lite optional-lite libsodium)
 if(NOT WIN32)
 	list(APPEND LIBMINIFI_LIBRARIES OSSP::libuuid++)
+else()
+	list(APPEND LIBMINIFI_LIBRARIES pdh)
```

Comment:
pdh (Performance Data Helper) is required to collect CPU usage information on windows